portfolio/src/main/java/com/google/sps/data/CommentBuilderImplementationLocal.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class CommentBuilderImplementationLocal implements CommentBuilderImplementation {
+    private Comment comment;
+
+    public CommentBuilderImplementationLocal() {
+        this.comment = new Comment();
+    }
+
+    @Override
+    public void  setId(long id) {
+        this.comment.setId(id);
+    }
+
+    @Override
+    public void setName(String name) {
+        this.comment.setName(name);
+    }
+
+    @Override
+    public void setDescription(String description) {
+        this.comment.setDescription(description);
+    }
+
+    @Override
+    public void setPostDate(long postDate) {
+        this.comment.setPostDate(postDate);
+    }
+
+    @Override
+    public void reset() {
+        this.comment = new Comment();
+    }
+
+    public Object build() {
```

Comment:
Because it inherits from an interface and the build function returns either a Comment (on local) or an Entity (on datastore). So, in order to adhere to the interface, I though of making it return an object. Is there a better work around this?

Reply:
It should always return the same type. Comment